docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -1,6 +1,6 @@
 [[host-isolation-ov]]
 [chapter, role="xpack"]
-= Host isolation
+= Isolate a host
```

Comment:
Not sure if this should remain as `Host isolation` as the paragraph begins by describing "host isolation". 🤔 Alternatively, we can re-word some of the below paragraph as following if we keep this change,

```md5
Isolating a host allows you to sever its connection from your network, blocking communication with other hosts on your network until you release the host. Isolating a host is useful for responding to malicious activity or preventing potential attacks, as it prevents lateral movement across other hosts.
```

Reply:
I think it's OK to mix up the phrasing a little for variety, since both phrases mean the same thing. I revised the heading in particular to make it more action-oriented (starting most headings with a verb), but we can also use noun forms throughout the paragraph.